src/app/components/book/book-list/book-list.component.html

```diff
@@ -0,0 +1,5 @@
+<h1>Books</h1>
+<div *ngFor="let book of books">
```

Comment:
Just for your information, no need to change it right now: it's good practise to add trackBy in ngFor, for better performance